internal/apiresource/service.go

```diff
@@ -387,17 +399,14 @@ func (d *Service) createRoute(service irtypes.Service) *okdroutev1.Route {
 			Labels: getServiceLabels(service.Name),
 		},
 		Spec: okdroutev1.RouteSpec{
-			Port: &okdroutev1.RoutePort{
-				TargetPort: intstr.IntOrString{
-					Type:   intstr.Int,
-					IntVal: common.DefaultServicePort,
-				},
-			},
+			Host: ir.TargetClusterSpec.Host,
```

Comment:
Do we need to specify host? If it takes default, we can ignore it. Isn't it?

Reply:
the default is a subdomain that uses the route name